### PR TITLE
typo in 0x0st.pl

### PR DESCRIPTION
**Pull request by pjalsGit**
indeed a 1 character change

### CODE REVIEW

**Comment by bw1** [comment]
this need a new `$VERSION = '0.04';` number :)

also this `changed	=> '2021-01-13',`

**Comment by pjalsGit** [comment]
> this need a new `$VERSION = '0.04';` number :)
> 
> also this `changed => '2021-01-13',`

i think i fixed it?

File: scripts/0x0st.pl [hunk 1]
```diff
@@ -8,15 +8,15 @@
...
+$VERSION = '0.05';
 %IRSSI = (
     authors	=> 'bw1',
     contact	=> 'bw1@aol.at',
     name	=> '0x0st',
     description	=> 'upload file to https://0x0.st/',
     license	=> 'ISC',
     url		=> 'https://scripts.irssi.org/',
-    changed	=> '2021-01-13',
+    changed     => '2021-09-28'
```
Comment (bw1) [inline scripts/0x0st.pl]:
``` 
     url―――⇥―――⇥=> 'https://scripts.irssi.org/',↵
-    changed―――⇥=> '2021-01-13',↵
+    changed     => '2021-09-28'↵
```

File: scripts/0x0st.pl [hunk 2]
```diff
@@ -10,16 +10,16 @@
...
+    authors	 => 'bw1',
+    contact	 => 'bw1@aol.at',
+    name	 => '0x0st',
+    description	 => 'upload file to https://0x0.st/',
+    license	 => 'ISC',
+    url		 => 'https://scripts.irssi.org/',
+    changed      => '2021-09-28'
+    modules      => 'POSIX HTTP::Request::Common LWP::UserAgent Storable File::Glob',
+    commands     => '0x0st',
+    selfcheckcmd => '0x0st -c',
```
Comment (bw1) [inline scripts/0x0st.pl:22]:
```
+    authors―――⇥ => 'bw1',↵
+    contact―――⇥ => 'bw1@aol.at',↵
+    name――⇥ => '0x0st',↵
+    description―――⇥ => 'upload file to https://0x0.st/',↵
+    license―――⇥ => 'ISC',↵
+    url―――⇥―――⇥ => 'https://scripts.irssi.org/',↵
+    changed      => '2021-09-28'↵
+    modules      => 'POSIX HTTP::Request::Common LWP::UserAgent Storable File::Glob',↵
+    commands     => '0x0st',↵
+    selfcheckcmd => '0x0st -c',↵
```

File: scripts/0x0st.pl [hunk 3]
```diff
@@ -10,12 +10,12 @@
...
-    name	 => '0x0st',
-    description	 => 'upload file to https://0x0.st/',
-    license	 => 'ISC',
-    url		 => 'https://scripts.irssi.org/',
+    authors      => 'bw1',
+    contact      => 'bw1@aol.at',
+    name         => '0x0st',
+    description  => 'upload file to https://0x0.st/',
+    license      => 'ISC',
+    url          => 'https://scripts.irssi.org/',
```
Comment (bw1) [inline scripts/0x0st.pl:18]:
```
-    authors―――⇥ => 'bw1',↵
-    contact―――⇥ => 'bw1@aol.at',↵
-    name――⇥ => '0x0st',↵
-    description―――⇥ => 'upload file to https://0x0.st/',↵
-    license―――⇥ => 'ISC',↵
-    url―――⇥―――⇥ => 'https://scripts.irssi.org/',↵
+    authors      => 'bw1',↵
+    contact      => 'bw1@aol.at',↵
+    name         => '0x0st',↵
+    description  => 'upload file to https://0x0.st/',↵
+    license      => 'ISC',↵
+    url          => 'https://scripts.irssi.org/',↵
```

not tabs no cry. <https://www.jwz.org/doc/tabs-vs-spaces.html>

File: scripts/0x0st.pl [hunk 4]
```diff
@@ -10,12 +10,12 @@
...
     changed      => '2021-09-28'
```
Comment (bw1) [inline scripts/0x0st.pl:19]:
```
syntax error at  0x0st.pl line 20, near "'2021-09-28'
```


```
    changed      => '2021-09-28',
                                ^
```

**Comment by bw1** [comment]
@pjalsGit

**Pull request closed without merging**